lighthouse-core/lib/url-shim.js

```diff
@@ -215,6 +215,30 @@ class URLShim extends URL {
     const urlScheme = protocol.includes(':') ? protocol.slice(0, protocol.indexOf(':')) : protocol;
     return NON_NETWORK_SCHEMES.includes(urlScheme);
   }
+
+  /**
+   * @param {string} src
+   * @return {string|undefined}
+   */
+  static guessMimeType(src) {
+    try {
+      const url = new URL(src);
+      if (url.protocol === 'data:') {
+        const match = url.pathname.match(/image\/(png|jpeg|jpg|svg\+xml|webp|gif|avif);/);
+        if (!match) return undefined;
+        return match[0].split(';')[0];
+      }
+
+      const match = src.toLocaleLowerCase().match(/\.(png|jpeg|jpg|svg|webp|gif|avif)$/);
+      if (!match) return undefined;
+
+      const ext = match[0].substr(1);
```

Comment:
```suggestion
      const ext = match[1];
```

since you called `toLocaleLowerCase` on src and already have a group